doc/loganalyzer_to_pytest.md

```diff
@@ -0,0 +1,74 @@
+#### Requirements
+Move existed loganalyzer to be supported by pytest framework.
+
+#### Overview
+At this approach all the work can be done on the Ansible host if script will download extracted syslog file to the Ansible host.
+
+Current loganalyzer implementation description:
+https://github.com/Azure/SONiC/wiki/LogAnalyzer
+
+#### Development
+Module "loganalyzer.py" with class "Loganalyzer". Class is intendent to do:
+- Add start marker to the DUT syslog

```

Comment:
How would the start marker be looked like? Should we consider haveing the start marker string contain testname? If so, testname need to be passed in to the __init__ method as an argument.

Reply:
Existed roles\test\files\tools\loganalyzer\loganalyzer.py will be reused, so logic with adding start/stop markers and processing the logs will be the same (will be reused). And yes, start marker is based on the test name.